src/pudl/package_data/phmsagas/column_maps/yearly_miles_of_transmission_pipe_by_nps.csv

```diff
@@ -20,7 +20,8 @@ onshore_transmission_pipe_nps_10_miles,,,,,,,,,,,,,,,,,,,,,parthon10,parthon10,p
 onshore_transmission_pipe_nps_12_miles,,,,,,,,,,,,,,,,,,,,,parthon12,parthon12,parthon12,parthon12,parthon12,parthon12,parthon12,parthon12,parthon12,parthon12,parthon12,parthon12,parthon12
 onshore_transmission_pipe_nps_14_miles,,,,,,,,,,,,,,,,,,,,,parthon14,parthon14,parthon14,parthon14,parthon14,parthon14,parthon14,parthon14,parthon14,parthon14,parthon14,parthon14,parthon14
 onshore_transmission_pipe_nps_16_miles,,,,,,,,,,,,,,,,,,,,,parthon16,parthon16,parthon16,parthon16,parthon16,parthon16,parthon16,parthon16,parthon16,parthon16,parthon16,parthon16,parthon16
-onshore_transmission_pipe_nps_18_miles,,,,,,,,,,,,,,,,,,,,,parthon18,parthon18,parthon18,parthon18,parthon18,parthon18,parthon18,parthon18,parthon18,parthon18,parthon18,parthon18,parthon18
+onshore_transmission_pipe_nps_18_miles,,,,,,,,,,,,,,,,,,,,,parthon18,parthon18,parthon18,parthon18,supplement
+,parthon18,parthon18,parthon18,parthon18,parthon18,parthon18,parthon18,parthon18
```

Comment:
oopsies this was a mistake and was causing the ci to fail.... removing and testing now